draft-ietf-lake-edhoc.md

```diff
@@ -1747,12 +1747,26 @@ In EDHOC, compact representation is used for the ephemeral public keys (G_X and
 
 The encoding of the point at infinity is not supported.
 
-Compact representation does not change any requirements on validation, see {{crypto}}. The following may be needed for validation or compatibility with APIs:
+Compact representation does not change any requirements on validation, see {{crypto}}. Using compact representation has some security benefits. An implementation does not need to check that the point is not the point at infinity (the identity element). Similarly, as not even the sign of the y-coordinate is encoded, compact representation trivially avoids so called "benign malleability" attacks where an attacker changes the sign, see {{SECG}}.
 
-* If a y-coordinate is required then the value ~yp SHALL be set to zero
-* The compact representation described above can in such a case be transformed into the SECG point compressed format by prepending it with the single byte 0x02 (i.e., M = 0x02 \|\| X).
+The following may be needed for validation or compatibility with APIs that do not support compact representation or do not support the full {{SECG}} format:
 
-Using compact representation has some security benefits. An implementation does not need to check that the point is not the point at infinity (the identity element). Similarly, as not even the sign of the y-coordinate is encoded, compact representation trivially avoids so called "benign malleability" attacks where an attacker changes the sign, see {{SECG}}.
+* If a compressed y-coordinate is required, then the value ~yp set to zero can be used. The compact representation described above can in such a case be transformed into the SECG point compressed format by prepending it with the single byte 0x02 (i.e., M = 0x02 \|\| X).
+* If a uncompressed y-coordinate is required, then a y-coordinate has to be calculated following Section 2.3.4 of {{SECG}} or Appendix C of {{RFC6090}}. Any of the square roots can be used.
+
+For example: The curve P-256 has the parameters (using the notation in {{RFC6090}})
+
+* p = 2^256 − 2^224 + 2^192 + 2^96 − 1
+* a = -3
+* b = 41058363725152142129326129780047268409114441015993725554835256314039467401291
+
+Given an example x
+
+* x = 115792089183396302095546807154740558443406795108653336398970697772788799766525
+
+we can calculate y = w = (x^3 + a * x + b)^((p + 1)/4) (mod p)
```

Comment:
Note somewhere that (p+1)/4 (mod p) is actually the square root operation, as explained in RFC6090.